### DISCUSSION TITLE
dual licensed?

### DISCUSSION THREAD

**Discussion by lyrra** [musescore/MuseScore#16615] [General]
Since Musescore4 is a derivative of Musescore3, isn't it dual licensed?
Ie, both GPL3 and GPL2.

**Comment by shoogle** [comment]
No, the code in general is not dual-licensed. However, bits of code that appear in both versions of the program are indeed dual licensed.

You are correct that when we "changed" the license, we couldn't undo the fact that MuseScore 3's code was already GPLv2, so by re-releasing same code under GPLv3 we were effectively dual-licensing that code GPLv2/GPLv3. However, when we then started editing that code, all the edits are under GPLv3 only, which means that MuseScore 4 as a whole is under GPLv3 only, even though specific bits that are unchanged from MuseScore 3 are also available under GPLv2.
